osu.Game.Tests/Visual/Multiplayer/TestMultiplayerGameplay.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using NUnit.Framework;
+using osu.Game.Screens.OnlinePlay.Multiplayer;
+
+namespace osu.Game.Tests.Visual.Multiplayer
+{
+    public class TestMultiplayerGameplay : MultiplayerTestScene
```

Comment:
I think this testscene is missing things, or is not working...

Reply:
This was... not supposed to be committed.